src/components/UI/organisms/MenuItem/MenuItem.css

```diff
@@ -0,0 +1,114 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@400;700&display=swap');
+
+:root {
+  --main: #fad390;
+  --secondary: #636e72;
+}
+
+.MenuItem {
```

Comment:
You are all aware of the fact you are using `sass` that allows `inside` nesting? Thanks to that you can `simulate` BEM `classes` with inside `nestings` max to fourth level depth 😄